.storybook/blocks/ComponentDetails.jsx

```diff
@@ -88,6 +88,44 @@ export const StatusLight = styled.span(({ variant = "positive", ...props }) => `
 	margin-block-end: 1px;
 `);
 
+export const ResourceSection = styled.section`
+	display: flex;
+	flex-flow: row wrap;
+	align-items: center;
+`;
+	
+export const ResourceLink = styled.a`
+	position: relative;
```

Comment:
I have two questions about the styles for the link cards that I'd like some opinions on.

- The resource cards on the docs site do have a background color, I believe because they're using `.spectrum-Card` styles. I didn't immediately add that because I wasn't sure on the whole light/mode dark dilemma I mentioned below, and didn't want to hard code anything yet. But should I specify a background color? (probably white?)

- Most of the font colors & styles are being passed down from the `.spectrum-Heading` and `.spectrum-Body` classes, but for some reason the underlay page isn't pulling those in. The styles for `.spectrum-Heading` and `.spectrum-Body` are undefined for the underlay page. I believe it's the only page that has blue link text, instead of black, and all of the font weights and sizes are inconsistent to the other component pages. To be safe, should I hard code any of those styles? I know underlay is funky component, so maybe it's implemented in a different way than our other components, so it isn't loading the heading and body styles at the correct time? I wasn't sure of another way to fix that page specifically, or the way to debug the missing styles.

Reply:
I think this is happening in Dialog too with the fonts, I have no idea why though:
![image](https://github.com/user-attachments/assets/f450af32-0480-4090-b71d-039f56d5c1db)

How many styles need to be added/hardcoded here to make those two components match the others?

Reply:
I didn't see this happening on dialog, but I did add styles for the font weight, color, font size, line-height and font-family. https://github.com/adobe/spectrum-css/pull/3129/commits/8fed17befd48f22212d97f921602a881e988266f